src/components/document-renderer.tsx

```diff
@@ -40,13 +54,30 @@ export const DocumentRenderer: React.FunctionComponent<DocumentRendererProps> =
     }
   }, [document, toFrame]);
 
+  useEffect(() => {
+    if (toFrame && selectedTemplate) {
+      // TODO: how can i get this to run? selected template is updated on new tab click but toFrame is null so it does not run.
+      toFrame({
```

Comment:
When I click on one of the tabs, selectedTemplate will be changed and i should run toFrame to update the template. But how can i get toFrame to not have a value of null? i don't really understand how toFrame is used